apps/condo/lang/ru/ru.json

```diff
@@ -2313,6 +2313,7 @@
   "tour.guide.aboutApp.extraIncome.card.body": "Приносит дополнительный доход от платных услуг",
   "tour.guide.aboutApp.extraIncome.modal.picText": "Приложение для Жителя приносит дополнительный доход от платных услуг",
   "tour.guide.aboutApp.extraIncome.modal.body": "Управляющая компания может предлагать жителям дополнительные услуги через приложение. Это приносит доход, а за счет автоматизации процессов затраты на обработку платных заявок снижаются. Качественный сервис, проверенные мастера, удобное оформление и оплата услуги повышает лояльность жителей.",
+  "tour.guide.aboutApp.extraIncome.modal.card.body": "Благодаря маркетплейсу УК «Авалон» предлагает жителям дополнительные платные услуги и создает финансовую подушку безопасности. За первый месяц работы через маркетплейс компания заработала 100 тысяч рублей, предоставляя всего шесть услуг.",
```

Comment:
What’s this? Where is en translation? And why some Avalone in open source translations ?